schema.graphql

```diff
@@ -235,7 +241,7 @@ type Query {
   allDatasets(orderBy: DatasetOrderBy = ORDER_BY_DATE,
               sortingOrder: SortingOrder = DESCENDING,
               filter: DatasetFilter = {}, simpleQuery: String,
-              offset: Int = 0, limit: Int = 10): [Dataset!]!
+              offset: Int = 0, limit: Int = 10, inpFdrLvls: [Float]): [Dataset!]!
```

Comment:
Somehow, the change in the function signature didn't affect its implementation. Forgot to remove?